src/axom/quest/examples/CMakeLists.txt

```diff
@@ -101,7 +216,8 @@ if(CONDUIT_FOUND)
             FOLDER      axom/quest/examples
             )
 
-    if(AXOM_ENABLE_TESTS AND AXOM_DATA_DIR)
+    # These examples currently segfault on windows without MPI when loading the test data into conduit
```

Comment:
😢

Reply:
I have the following comment below:

> @gunney1 -- I fixed a bug in the non-mpi version of the marching cubes example when testing on Windows.

> Unfortunately, I couldn't get this example to run on Windows -- where axom and conduit were built without MPI. I am not sure if this was because of the multidomain setup or the use of a different hdf5 (1.12 vs. 1.8). In the meantime, I disabled the tests of this example on Windows.

Also, the expected input uses multi-domain blueprints and was generated by a python script (in `axom/src/tools`) that uses conduit's python interface, which I didn't have in my windows configuration.

The more direct answer is that there were a lot of moving parts (MPI vs. no MPI; different versions of hdf5; windows vs. linux) and getting these tests running on Windows was out of scope for this PR. I did fix the build though.

I'll create an issue for us to address this in the future.